content/src/display-content/business-formation/nexus/dba-resolution-foreign.md

```diff
@@ -4,14 +4,12 @@ filename: dba-resolution-foreign
 displayname: dba-resolution-foreign
 name: Get a Resolution Adopting Your DBA Name
 id: DBA-resolution-foreign
+summaryDescriptionMd: >-
+  To authorize your business in New Jersey, you need to obtain a resolution adopting your Doing Business As
+  (DBA) name.
+bodyHeading: How to Get a Resolution
 ---
```

Comment:
[dust]: Would this create different page content? With the previous ones the body header was above the summaryDescription but with this one it was below in the diff.